src/prefect/utilities/compatibility.py

```diff
@@ -21,3 +20,18 @@ def nullcontext() -> Iterator[None]:
 else:
 
     from contextlib import nullcontext  # noqa: F401
+
+
+# Provide 3.6/3.7 `call` with `.args` and `.kwargs`
+
+
+def Call(call):  # type: ignore
```

Comment:
Since this is only used in a helper function in a few tests in one file, it doesn't really feel worth it to me to pull it out into a separate file. Honestly, I'd just keep indexing the `call_args` tuple rather than making a helper like this (we do the indexing everywhere in our tests, consistency matters). When we drop support for older Python versions we can use the attribute accessors.